_commands/tools/gdb.md

```diff
@@ -0,0 +1,82 @@
+---
+---
+
+GDB
+-------
+
+An overview on the GNU debugger GDB.  
+
+~~~ bash
+*short* example of command usage and output
+~~~
+
+<!--more-->
+
+
+## Starting GDB
+
+Starting GDB is simple! When you are compiling your C++ or C code, change your optimization
+flag to -g.
+
+`g++ -Wall -Werror -g -std=c++11 testprogram.cpp -o testprogram.exe`
+
+Then, after compiling the program, run the program.
+
+`gdb testprogram.exe`
+
+If gdb shows the error 'No symbol table is loaded' then, -g could be missing.
+
+
+## Beginning to run your program
+
+Just start your program by pressing start. Then, gdb will set a breakpoint at the beginning
+of int main().  
+
+`start`
```

Comment:
Do you mean `run`? Also, could you include what the shortcuts for these commands are (for example, it would be `r` in this case).